refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -246,27 +223,20 @@ private Problem reallyUndoRefactoring(boolean saveAfterDone) {
                     f.undoChange();
                 }
             }
-            for (Transaction commit : SPIAccessor.DEFAULT.getCommits(bag)) {
-                SPIAccessor.DEFAULT.check(commit, true);
-            }
-            UndoableWrapper wrapper = MimeLookup.getLookup("").lookup(UndoableWrapper.class);
+            SPIAccessor.DEFAULT.getCommits(bag).forEach(commit -> SPIAccessor.DEFAULT.check(commit, true));
+            UndoableWrapper undoableWrapper = MimeLookup.getLookup("").lookup(UndoableWrapper.class);
```

Comment:
// NOI18N marker missing.

Reply:
There is no string, which needs to be translated. Why adding a comment?

Reply:
The `// NOI18N` indicates that the string does not need to be translated.

I believe that some devs have a plugin which places warnings on all strings that are not marked "NOI18N" as a way of verifying that all strings either do not need translation or are localizable.